tools/mo/openvino/tools/mo/main.py

```diff
@@ -138,11 +138,27 @@ def get_moc_frontends(argv: argparse.Namespace):
 
 
 def arguments_post_parsing(argv: argparse.Namespace):
+    use_legacy_frontend = argv.use_legacy_frontend
+    use_new_frontend = argv.use_new_frontend
+
+    if use_new_frontend and use_legacy_frontend:
+        raise Error('Options --use_new_frontend and --use_legacy_frontend must not be used simultaneously '
+                    'in the Model Optimizer command-line')
+
     moc_front_end, available_moc_front_ends = get_moc_frontends(argv)
 
+    if not moc_front_end and use_new_frontend:
+        raise Error('Option --use_new_frontend is specified but the Model Optimizer is unable to find new frontend. '
+                    'Please ensure that your environment contains new frontend for the input model format.')
+
     is_tf, is_caffe, is_mxnet, is_kaldi, is_onnx =\
```

Comment:
what happens if new_frontend is not found and we can't deduce the framework by namespace? 
do we print informative message?

Reply:
Great catch to check!

Reply:
Handle a default case when frontend is not found and recommend a list of frameworks from which user can select